_tests/e2e/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM test-base:2.3.0
 
 WORKDIR /tmp
-RUN yum install -y bzip2 fontconfig freetype freetype-devel fontconfig-devel libstdc++ && \
+RUN yum install -y bzip2 fontconfig freetype freetype-devel fontconfig-devel libstdc++ unzip && \
```

Comment:
Ideally, this would include a `&& yum clean all`

Reply:
Same thing here.

Reply:
We'd also want to remove the downloaded tarball at the end of this string of commands.